.evergreen/config.yml

```diff
@@ -24116,6 +24116,112 @@ tasks:
       AUTH: noauth
       LOADBALANCED: loadbalanced
       SSL: nossl
+- name: testazurekms-task
+  commands:
+  - func: fetch source
+  - command: shell.exec
+    params:
+      shell: bash
+      script: |-
+        set -o errexit
+        echo "Building test-azurekms ... begin"
+        pushd mongoc
+        ./.evergreen/compile-test-azurekms.sh
+        popd
+        echo "Building test-azurekms ... end"
+        echo "Copying files ... begin"
+        export AZUREKMS_RESOURCEGROUP=${testazurekms_resourcegroup}
+        export AZUREKMS_VMNAME=${AZUREKMS_VMNAME}
+        export AZUREKMS_PRIVATEKEYPATH=/tmp/testazurekms_privatekey
+        DRIVERS_TOOLS=$(pwd)/drivers-evergreen-tools
```

Comment:
NOTE: if you use DRIVERS_TOOLS, this value should already be assigned, at least this is what I saw in the c# driver (same in other places)

Reply:
Missed your explanation above. It's a bit strange, but ok :)